client/src/components/reviewsSubComponents/ReviewTile.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import RatingStars from 'react-star-ratings';
+
+const TileContainer = styled.div`
+  margin: 0 auto;
+  padding: 10px 30px;
+  border-bottom: 1px solid black;
+`;
+
+const monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
```

Comment:
You should move this variable down so that your styled components and react elements are grouped